Gems/Atom/RPI/Code/Source/RPI.Edit/Shader/ShaderSourceData.cpp

```diff
@@ -133,28 +158,6 @@ namespace AZ
                     ->Constructor<const ShaderSourceData::ProgramSettings&>()
```

Comment:
Should the `ShaderSourceData::ProgramSettings` also be moved before `ShaderSourceData` which has a `ShaderSourceData::m_programSettings` property?

Reply:
I could, but I did compile and run this, and it did function without issue.